ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422__model_serving_llm_UI.robot

```diff
@@ -267,7 +267,7 @@ Verify User With Edit Permission Can Deploy Query And Delete A LLM
     [Documentation]    This test case verifies that a user with Edit permission on a DS Project can still deploy, query
     ...    and delete a LLM served with caikit
     ...    ProductBug: https://issues.redhat.com/browse/RHOAIENG-548
```

Comment:
@asanzgom could you please delete the Productbug links from the doc of the test case?

Reply:
this has been fixed in 2.6 already... wonder why it wasn't been removed from known issues that time...

Regarding the links to the reported issues - I slightly keen to keep them here for reference, just to remove the `ProductBug` at the beginning. Something like:

```suggestion
    ...    Issue reporated for this test in the past: https://issues.redhat.com/browse/RHOAIENG-548
```

but I can live with removal also

Reply:
sure I will